bitnami/phpmyadmin/templates/deployment.yaml

```diff
@@ -103,11 +103,17 @@ spec:
             {{- if .Values.extraEnvVars }}
             {{- include "phpmyadmin.tplValue" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
             {{- end }}
-          {{- if .Values.extraEnvVarsCM }}
+          {{- if or .Values.extraEnvVarsCM .Values.extraEnvVarsSecret }}
```

Comment:
You can remove this if statement. In other charts we don't have it. The worst that will happen will be an empty envFrom.